observability/metrics/src/test/suite/scala/com/daml/metrics/InstrumentedGraphSpec.scala

```diff
@@ -61,9 +68,9 @@ final class InstrumentedGraphSpec extends AsyncFlatSpec with Matchers with AkkaB
     val lowAcceptanceThreshold = bufferSize - acceptanceTolerance
     val highAcceptanceThreshold = bufferSize + acceptanceTolerance
 
-    val maxBuffered = DropwizardCounter("test-length", new InstrumentedGraphSpec.MaxValueCounter())
-    val capacityCounter = DropwizardCounter("test-capacity", new codahale.Counter())
-    val delayTimer = DropwizardTimer("test-delay", new codahale.Timer())
+    val maxBuffered = InMemoryCounter(MetricsContext.Empty)
+    val capacityCounter = InMemoryCounter(MetricsContext.Empty)
```

Comment:
This test fails with an `IllegalArgumentException`.
`MetricValues.CounterValues.value` is missing support for `InMemoryCounter`.

Reply:
Fixed in the latest commit